initramfs/scripts/bootloader-script

```diff
@@ -0,0 +1,190 @@
+# -*- shell-script -*- vim:ft=sh:
```

Comment:
Can you add this to the script or to a readme file? This is going to be pretty opaque to others without documentation